tests/CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ set(LVGL_TEST_OPTIONS_TEST_SYSHEAP
 
 set(LVGL_TEST_OPTIONS_TEST_DEFHEAP
     -DLV_TEST_OPTION=5
+    -DLV_USE_OBJ_PROPERTY=1      # add obj property test and disable pedantic
```

Comment:
Can it work for sysheap too?

Reply:
Yes, but it will generate warning regarding pedantic. So only one of the test option turns on `LV_USE_OBJ_PROPERTY` and disable the warning correspondingly.